src/controllers/user.ts

```diff
@@ -79,7 +82,79 @@ const userController = {
         if (!errors.isEmpty()) {
             res.status(statusCodes.MISSING_PARAMS).json(errors.formatWith(errorMessage).array()[0]);
         } else {
-            res.status(statusCodes.SUCCESS).send({msg: "Not implemented"});
+            try {
+                const { userId } = req.params;
+                const user: IUserModel = await userDBInteractions.find(userId);
+                if (!user)
+                    res.status(statusCodes.NOT_FOUND).send({ status: statusCodes.NOT_FOUND, message: "User not found" });
+                else {
+                    const userObject: IUser = {
+                        email: user.email,
+                        githubRepo: user.githubRepo,
+                        githubToken: user.githubToken,
+                        githubUsername: user.githubUsername,
+                        scores: user.scores
+                    };
+
+                    const updatedUserBody: IUser = {
+                        ...userObject,
+                        ...req.body
```

Comment:
add `scores: user.score` here since this route shouldn't be able to update the score (only the update score route should do that)